### PR TITLE
TW-1796: Standarlize Appbar and Appgrid popup

### PR DESCRIPTION
**Pull request by hieutbui**
## Ticket
- #1796 

## Resolved

https://github.com/linagora/twake-on-matrix/assets/80142234/b5cf7cac-6d14-46ee-b80a-db5ead96e61b

### CODE REVIEW

**Comment by nqhhdev** [comment]
IMO: We keep all appgird icons, only update `configurations/app_dashboard.json` file.
- [ ] Remove if not used
- [ ] Update ADR `docs/configurations/app_grid_configuration.md`

File: lib/pages/app_grid/app_grid_dashboard_controller.dart [hunk 1]
```diff
@@ -50,14 +52,33 @@ class AppGridDashboardController extends State<AppGridDashboard> {
...
+      'Twake Mail': true,
+      'Twake Drive': true,
+    };
+
+    final activeApps = linagoraApps.apps
+        .where((app) => mapActiveApps[app.appName] == true)
+        .toList();
+
+    return LinagoraApplications(activeApps);
+  }
```
Comment (nqhhdev) [inline lib/pages/app_grid/app_grid_dashboard_controller.dart:80]: No need

File: lib/pages/app_grid/app_grid_dashboard_item.dart [hunk 2]
```diff
@@ -28,37 +33,70 @@ class AppGridDashboardItem extends StatelessWidget {
...
+                              width: AppGridDashboardItemStyle.iconAppSize,
+                              height: AppGridDashboardItemStyle.iconAppSize,
+                              fit: BoxFit.fill,
+                            ),
+                    Padding(
+                      padding: AppGridDashboardItemStyle.textPadding,
+                      child: Text(
+                        app.appName.contains('Twake')
+                            ? app.appName.split('Twake ').last
+                            : app.appName,
```
Comment (nqhhdev) [inline lib/pages/app_grid/app_grid_dashboard_item.dart:88]: IMO: Create an extension for handle this case.

File: lib/pages/app_grid/app_grid_dashboard_item_style.dart [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+
+  static const BorderRadius borderRadius =
+      BorderRadius.all(Radius.circular(10));
+
+  static TextStyle? textStyle(BuildContext context) =>
+      Theme.of(context).textTheme.bodySmall?.copyWith(
+            fontSize: 12,
+            fontWeight: FontWeight.w400,
+            color: Colors.black,
+          );
```
Comment (nqhhdev) [inline lib/pages/app_grid/app_grid_dashboard_item_style.dart:21]: - Why need to use `foneSize` and `fontWeight`?
- Should use the color of `Linagora system design`

File: lib/pages/app_grid/app_grid_dashboard_controller.dart [hunk 4]
```diff
@@ -50,14 +52,33 @@ class AppGridDashboardController extends State<AppGridDashboard> {
...
+    LinagoraApplications linagoraApps,
+  ) {
+    final Map<String, bool> mapActiveApps = {
+      'Twake Chat': false,
+      'Contacts': false,
+      'Calendar': false,
+      'Teleskop': false,
+      'Twake Mail': true,
+      'Twake Drive': true,
+    };
```
Comment (sherlockvn) [inline lib/pages/app_grid/app_grid_dashboard_controller.dart:73]: i think you can make a contant, and may be enum is better

File: lib/pages/app_grid/app_grid_dashboard_overlay.dart [hunk 5]
```diff
@@ -11,10 +12,10 @@ class AppGridDashboardOverlay extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      margin: AppGridDashboardOverlayStyle.margin,
       width: AppGridDashboardOverlayStyle.widthAppGrid(_linagoraApplications),
       decoration: BoxDecoration(
-        color: Theme.of(context).colorScheme.surface,
+        color: LinagoraSysColors.material().onPrimary,
```
Comment (sherlockvn) [inline lib/pages/app_grid/app_grid_dashboard_overlay.dart:17]: why don you use `Theme.of....`

File: docs/configurations/app_grid_configuration.md [hunk 6]
```diff
@@ -15,6 +15,7 @@
       "appName": "Twake Chat",
       "icon": "ic_twake_chat.svg",
       "appLink": "https://beta.twake.app/",
+      "publicIconUri": "xxx",
```
Comment (sherlockvn) [inline docs/configurations/app_grid_configuration.md:18]: i think you should add an TO-DO

File: lib/pages/app_grid/app_grid_dashboard_controller.dart (hunk 1 above)
Comment (hieutbui) [reply lib/pages/app_grid/app_grid_dashboard_controller.dart]: Removed

File: lib/pages/app_grid/app_grid_dashboard_item.dart (hunk 2 above)
Comment (hieutbui) [reply lib/pages/app_grid/app_grid_dashboard_item.dart]: Done

File: lib/pages/app_grid/app_grid_dashboard_item_style.dart (hunk 3 above)
Comment (hieutbui) [reply lib/pages/app_grid/app_grid_dashboard_item_style.dart]: Removed cause no need

File: lib/pages/app_grid/app_grid_dashboard_controller.dart (hunk 4 above)
Comment (hieutbui) [reply lib/pages/app_grid/app_grid_dashboard_controller.dart]: Updated

File: lib/pages/app_grid/app_grid_dashboard_overlay.dart (hunk 5 above)
Comment (hieutbui) [reply lib/pages/app_grid/app_grid_dashboard_overlay.dart]: Done

File: docs/configurations/app_grid_configuration.md (hunk 6 above)
Comment (hieutbui) [reply docs/configurations/app_grid_configuration.md:18]: This is config docs so I think no need TO-DO

**Review by Te-Z — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]
Tested on WEB (Safari, chrome, Arc, Firefox) works well.

**Review by Te-Z — APPROVED** [review]

**Pull request merged**